src/commandDetails/admin/ban.ts

```diff
@@ -31,11 +31,9 @@ const banExecuteCommand: SapphireMessageExecuteType = async (client, messageFrom
       );
     }
     const days = <number>args['days'];
-    // Get the GuildMember object corresponding to the user in the guild
-    // This is needed because we can only ban GuildMembers, not Users
+    // get Guild object corresponding to server
     const guild = await client.guilds.fetch(vars.TARGET_GUILD_ID);
-    const memberInGuild = await guild.members.fetch({ user });
-    if (await banUser(memberInGuild, reason, days)) {
+    if (await banUser(guild, user, reason, days)) {
       return `Successfully banned user ${user.tag} (id: ${user.id}) ${
         days ? `and deleted their messages in the past ${days} days ` : ``
```

Comment:
Here, just update `days` to `${pluralize('day', days)}` like in `src/components/admin.ts/makeBanMessage`